pkg/obfuscate/sql.go

```diff
@@ -18,7 +18,11 @@ var questionMark = []byte("?")
 
 // discardFilter is a token filter which discards certain elements from a query, such as
 // comments and AS aliases by returning a nil buffer.
-type discardFilter struct{ keepSQLAlias bool }
+type discardFilter struct {
+	keepSQLAlias    bool
+	collectComments bool
```

Comment:
This seems to use `keep` and `collect` as equivalent -- would it make sense to stick to one term?

Reply:
Thanks for the review! I think `collect` makes more sense in this case because comments will be removed from the query and `keep` indicates that the comments will remain in the query after obfuscation.

Reply:
Ah, nice distinction :)